src/hooks/v2/nftRewards/NFTRewardsToIPFS.ts

```diff
@@ -1,6 +1,6 @@
 import { NFTRewardTier } from 'models/v2/nftRewardTier'
 
-type CloudFunctionRewardTier = {
+export type CloudFunctionRewardTier = {
```

Comment:
**suggestion** we might find a different name for this. This cfn will live in-app on NextJS eventually.

Reply:
Shouldnt need this now that we're not using cloud function to upload. Will get rid of this in a follow up